src/components/main/list/SelectionContext.tsx

```diff
@@ -0,0 +1,130 @@
+import {
+  createContext,
+  useCallback,
+  useContext,
+  useMemo,
+  useRef,
+  useState,
+} from 'react';
+
+import { PRIMARY_COLOR } from '@graasp/ui';
+
+import {
+  Box,
+  boxesIntersect,
+  useSelectionContainer,
+} from '@air/react-drag-to-select';
+
+import { ITEM_CARD_CLASS } from '@/config/selectors';
+
+type SelectionContextValue = {
+  selectedIds: string[];
+  toggleSelection: (_el: string) => void;
```

Comment:
Not sure to understand what is the `el` of type string. Also why using the `_` ? If it is never used, just remove it ?